test/ThumbnailSpec.js

```diff
@@ -5,17 +5,24 @@ import ReactTestUtils from 'react-dom/test-utils';
 import Thumbnail from '../src/Thumbnail';
 
 describe('<Thumbnail>', () => {
+  let defaultProps = {
```

Comment:
probably it was fine without this `defaultProps`? worth verifying that things still work if you don't specify all of these, like we were implicitly doing before